modules/validations/missing_tag.js

```diff
@@ -43,7 +43,7 @@ export function validationMissingTag(context) {
         var subtype;
 
         var osm = context.connection();
-        var isUnloadedNode = entity.type === 'node' && osm && !osm.isDataLoaded(entity.loc);
+        var isUnloadedNode = entity.type === 'node' && osm && typeof osm.isDataLoaded === 'function' && !osm.isDataLoaded(entity.loc);
```

Comment:
I wonder if it would be better to call `context.validator().reset()` in an `afterEach` block. That function calls `cancelIdleCallback`, thereby addressing the root cause. This reset command could possibly be added to [the spec_helpers file](https://github.com/openstreetmap/iD/blob/develop/test/spec_helpers.ts), so that it applies to every test. 

Likewise for combo: we could call `uiCombobox.off()`, which is a global singleton that cancels any pending `setTimeout`s.

---

i tried this locally, but since it's an intermittent issue, i also don't know if it truely solves the problem 😅